### PR TITLE
utils_net:Update function get_linux_iface_info

### PR DESCRIPTION
**Pull request by chloerh**
To support getting interface info of given interface

### CODE REVIEW

**Comment by chloerh** [comment]
Before:
 (1/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link: ERROR: 'NoneType' object has no attribute 'get' (54.58 s)
 (2/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_with_rom: ERROR: 'NoneType' object has no attribute 'get' (45.58 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_without_addr: ERROR: 'NoneType' object has no attribute 'get' (45.56 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_diff_type: ERROR: 'NoneType' object has no attribute 'get' (57.48 s)

After:
 (1/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link: PASS (55.14 s)
 (2/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_with_rom: PASS (46.09 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_without_addr: PASS (45.60 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtual_network.iface_update.positive_test.update_link_diff_type: PASS (57.66 s)

**Review by Yingshun — APPROVED** [review]
LGTM

File: virttest/utils_net.py [hunk 1]
```diff
@@ -3722,31 +3722,35 @@ def _process_output(cmd, reg_pattern):
                                "mac %s" % mac_address)
 
 
-def get_linux_iface_info(mac, session=None):
+def get_linux_iface_info(iface='', mac=None, session=None):
     """
     Get host/guest info of certain interface with given mac address.
 
     :param mac: string type, mac address of given interface
```
Comment (dzhengfy) [inline virttest/utils_net.py:3729]: Please add docstring for new parameter

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: virttest/utils_net.py (hunk 1 above)
Comment (chloerh) [reply virttest/utils_net.py]: Done

**Review by dzhengfy — APPROVED** [review]

**Pull request merged**